test/Microsoft.TestPlatform.AcceptanceTests/Microsoft.TestPlatform.AcceptanceTests.csproj

```diff
@@ -22,7 +22,7 @@
       <Version>$(ChutzpahAdapterVersion)</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.TestPlatform.TestAsset.NativeCPP">
-      <Version>1.0.0</Version>
+      <Version>2.0.0</Version>
```

Comment:
Please make sure we are creating Nuget package of Microsoft.TestPlatform.TestAsset.NativeCPP with version 2.0.0